cg/cli/upload/clinical_delivery.py

```diff
@@ -104,14 +105,24 @@ def auto_fastq(context: click.Context, dry_run: bool):
                 )
                 analysis_obj.uploaded_at = dt.datetime.now()
             else:
-                LOG.warning(
+                LOG.debug(
                     "Upload to clinical-delivery for %s has already started, skipping",
                     analysis_obj.family.internal_id,
                 )
             continue
         case: Family = analysis_obj.family
         LOG.info("Uploading family: %s", case.internal_id)
         analysis_obj.upload_started_at = dt.datetime.now()
-        context.invoke(upload_clinical_delivery, case_id=case.internal_id, dry_run=dry_run)
+        try:
+            context.invoke(upload_clinical_delivery, case_id=case.internal_id, dry_run=dry_run)
+        except Exception as error:
+            LOG.error(f"Upload of case {case.internal_id} failed")
+            LOG.error(error)
+            exit_code: int = EXIT_FAIL
+            continue
+
         if not dry_run:
             status_db.session.commit()
```

Comment:
I would also like to add that with this general commit, we could introduce errors with this change, if we commit changes for a failed upload. I suggest we either do a more specific error handling and do bot continue regardless of the Exception raised, or that we create a transaction for each case and either do multiple commits, or rollback transactions that fail and keep the "large" commit.

Reply:
Looking at the command we are calling, there are no changes to any ORM objects. But it's true that it the pattern is very prone to bugs. SA suggested we review it when looking at the session management with the latest SQL-alchemy version

Reply:
Okay! Then I consider this comment to be out of scope :)